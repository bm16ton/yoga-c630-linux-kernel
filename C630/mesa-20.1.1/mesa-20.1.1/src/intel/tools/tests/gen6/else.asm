else(8)         JIP: 12                                         { align1 1Q };
else(16)        JIP: 12                                         { align1 1H };
else(8)         JIP: 18                                         { align16 1Q };
