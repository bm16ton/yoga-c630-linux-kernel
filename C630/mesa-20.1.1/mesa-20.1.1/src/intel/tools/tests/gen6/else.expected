24 00 60 00 8f 10 0c 00 00 00 8d 00 00 00 8d 00
24 00 80 00 8f 10 0c 00 00 00 8d 00 00 00 8d 00
24 01 60 00 8f 10 12 00 04 00 6e 00 04 00 6e 00
